.github/workflows/tests.yml

```
@@ -0,0 +1,27 @@
+name: Python package
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.8]
```

Not required, but it'll save us a plugin and provides the most flexibility 
```suggestion
      matrix:
        include:
          - python-version: 3.5
            tox-env: py35-django22
          - python-version: 3.8
            tox-env: py38-django22
```